<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MillerWidget</class>
 <widget class="QWidget" name="MillerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="7">
    <widget class="QLabel" name="lLabel">
     <property name="text">
      <string>&amp;l:</string>
     </property>
     <property name="buddy">
      <cstring>lSel</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="idxLabel">
     <property name="text">
      <string>Index:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QSpinBox" name="lSel"/>
   </item>
   <item row="3" column="7">
    <widget class="QDoubleSpinBox" name="zOff"/>
   </item>
   <item row="4" column="0" colspan="8">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Show/Hide</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="8">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>381</width>
       <height>180</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="xOff"/>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="hSel"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="hLabel">
     <property name="text">
      <string>h:</string>
     </property>
     <property name="buddy">
      <cstring>hSel</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="kSel"/>
   </item>
   <item row="3" column="4">
    <widget class="QDoubleSpinBox" name="yOff"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="kLabel">
     <property name="text">
      <string>&amp;k:</string>
     </property>
     <property name="buddy">
      <cstring>kSel</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="offLabel">
     <property name="text">
      <string>Offset:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hSel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MillerWidget</receiver>
   <slot>updateIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kSel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MillerWidget</receiver>
   <slot>updateIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lSel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MillerWidget</receiver>
   <slot>updateIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xOff</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MillerWidget</receiver>
   <slot>updateOffset(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yOff</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MillerWidget</receiver>
   <slot>updateOffset(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zOff</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MillerWidget</receiver>
   <slot>updateOffset(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateIndex(int)</slot>
  <slot>updateOffset(double)</slot>
  <slot>toggleView()</slot>
 </slots>
</ui>
